### PR TITLE
add info for by/without

### PR DESCRIPTION
**Pull request by Nexucis**
Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

**Comment by Nexucis** [comment]
will be nice to add a description for `on` / `ignoring` / `group_left` / `group_right`. But so far I didn't find a short and understable description

File: src/lang-promql/complete/promql.terms.ts [hunk 1]
```diff
@@ -320,7 +320,16 @@ export const aggregateOpTerms = [
...
 
-export const aggregateOpModifierTerms = [{ label: 'by' }, { label: 'without' }];
+export const aggregateOpModifierTerms = [
+  {
+    label: 'by',
+    info: 'Drop labels that are not listed in the "by" clause, even if their label values are identical between all elements of the vector',
+  },
+  {
+    label: 'without',
+    info: 'Remove the listed labels from the result vector, while all other labels are preserved the output.',
```
Comment (juliusv) [inline src/lang-promql/complete/promql.terms.ts:330]:
```suggestion
    info: 'Remove the listed labels from the result vector, while all other labels are preserved in the output.',
```

Although I would maybe even keep it simpler:

```suggestion
    info: 'Remove the listed labels, preserve all others.',
```

File: src/lang-promql/complete/promql.terms.ts [hunk 2]
```diff
@@ -320,7 +320,16 @@ export const aggregateOpTerms = [
   },
 ];
 
-export const aggregateOpModifierTerms = [{ label: 'by' }, { label: 'without' }];
+export const aggregateOpModifierTerms = [
+  {
+    label: 'by',
+    info: 'Drop labels that are not listed in the "by" clause, even if their label values are identical between all elements of the vector',
```
Comment (juliusv) [inline src/lang-promql/complete/promql.terms.ts:326]: We could make this one simpler too:

```suggestion
    info: 'Keep the listed labels, remove all others.',
```

**Comment by Nexucis** [comment]
ah yeah it's simple.
Thanks a lot ! I was definitely searching a much more complicated description hahaha.

**Comment by juliusv** [comment]
:+1: Nice, thanks! Another nice thing to add descriptions for would be our snippets (at some point) :)

**Comment by Nexucis** [comment]
ah yeah that's true ! and also maybe for `and` / `or` / `unless`

**Pull request merged**